modules/monitoring-understanding-alert-routing-for-user-defined-projects.adoc

```diff
@@ -7,16 +7,24 @@
 = Understanding alert routing for user-defined projects
 
 [role="_abstract"]
-As a cluster administrator, you can enable alert routing for user-defined projects. After doing so, you can allow users to configure alert routing for their user-defined projects. Users can then create and configure user-defined alert routing by creating or editing the `AlertmanagerConfig` objects.
+As a cluster administrator, you can enable alert routing for user-defined projects. 
+With this feature, you can allow users with the **alert-routing-edit** role to configure alert notification routing and receivers for user-defined projects.
+These notifications will then be routed using either the default Alertmanager instance or the optional Alertmanager instance dedicated only to user-defined monitoring, if enabled.
```

Comment:
```suggestion
These notifications are routed by the default Alertmanager instance or, if enabled, by an optional Alertmanager instance dedicated to user-defined monitoring.
```
- Avoids future tense.
- Avoids retrospectively figuring out what "if enabled" applies to.
- Doesn't avoid passive voice.

Reply:
Edited to read `These notifications are routed by the default Alertmanager instance or, if enabled, an optional Alertmanager instance dedicated to user-defined monitoring.` Much better!